ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/inmemory/InMemoryLedger.scala

```diff
@@ -75,6 +75,12 @@ class InMemoryLedger(
     l
   }
 
+  private val allocationEntries = {
+    val l = new LedgerEntries[PartyAllocationLedgerEntry](_.toString)
```

Comment:
hmm this will need some generalizing as the LedgerEntry type has a bunch of stuff that is not relevant for the PackageUpload and PartyAllocation entries.  Let's chat on this tomorrow

Reply:
this is now addressed through @dajmaki change to have `InMemoryPartyEntry` and `InMemoryPackageEntry` extend `InMemoryEntry` a la InMemoryLedgerEntry and use the same ledgerEntries ImmArray to store items, preserving ordering and offset